### PR TITLE
Another kitten

### PR DESCRIPTION
**Pull request by lpil**

### CODE REVIEW

File: lib/build_monitor.rb [hunk 1]
```diff
@@ -26,6 +23,25 @@ def config
...
+  def random_bad_image
+    [
+      "http://placekitten.com/300/#{301 + rand(10)}",
+      "http://1-ps.googleusercontent.com/h/www.catgifpage.com/gifs/224.gif.pagespeed.ce.8Ox__cf1NE.gif",
+      "http://1-ps.googleusercontent.com/h/www.catgifpage.com/gifs/223.gif.pagespeed.ce.74u-hdg7bt.gif",
+      "http://stream1.gifsoup.com/view/437149/roomba-kittens-o.gif",
+    ].sample
+  end
+
+  def random_bad_image
```
Comment (davidsevcik) [inline lib/build_monitor.rb]: random_bad_image twice?

Comment (errm) [inline lib/build_monitor.rb]: why are we defining `random_bad_image` twice

Comment (lpil) [inline lib/build_monitor.rb]: I have no idea what you mean.

File: lib/build_monitor.rb [hunk 2]
```diff
@@ -26,6 +23,25 @@ def config
   @config ||= YAML.load(File.read(config_file_name))
 end
 
+helpers do
+  def random_bad_image
+    [
+      "http://placekitten.com/300/#{301 + rand(10)}",
+      "http://1-ps.googleusercontent.com/h/www.catgifpage.com/gifs/224.gif.pagespeed.ce.8Ox__cf1NE.gif",
```
Comment (errm) [inline lib/build_monitor.rb]: This url does not seem to work

File: lib/build_monitor.rb [hunk 3]
```diff
@@ -26,6 +23,25 @@ def config
...
+      "http://1-ps.googleusercontent.com/h/www.catgifpage.com/gifs/223.gif.pagespeed.ce.74u-hdg7bt.gif",
```
Comment (errm) [inline lib/build_monitor.rb]: nor this

File: lib/build_monitor.rb [hunk 4]
```diff
@@ -26,6 +23,25 @@ def config
...
+      "http://stream1.gifsoup.com/view/437149/roomba-kittens-o.gif",
+    ].sample
+  end
+
+  def random_bad_image
+    [
+      "http://laughingthroughthepain.files.wordpress.com/2011/03/crazy-cat.gif",
+      "http://i.imgur.com/C9EtF56.gif",
+      "http://i.imgur.com/tAgFfQa.gif",
```
Comment (errm) [inline lib/build_monitor.rb]: Good choice with this one :)

File: lib/build_monitor.rb (hunk 1 above)
Comment (errm) [inline lib/build_monitor.rb]: your code is of the form

```ruby
helpers do
  def random_bad_image
    #kitten goodness
  end

  def random_bad_image
    #kitten badness
  end
end
```

So the second definition blats the first.

**Comment by jonnyarnold** [comment]
LGTMKK
(Looks Good To Merge. Kitty Kitty.)

File: lib/build_monitor.rb (hunk 4 above)
Comment (Antti) [inline lib/build_monitor.rb]: I actually like racoon's one the best.

**Comment by lpil** [comment]
I clearly have a lot more kitten work to do here. Will be busy with this project for the rest of my notice period.

**Comment by reneklacan** [comment]
:+1:

**Comment by jonnyarnold** [comment]
LGTMKK.

**Pull request merged**